client/src/lib/Sources/SourceCreator.svelte

```diff
@@ -93,6 +93,9 @@
 
   const saveAll = async () => {
     updateSourceForm();
+    if (source.age == "") {
```

Comment:
nit: Use `===` for comparison, unless there is a specific reason for it.

Reply:
Done